CONTRACTS_V2_LOG.md

```diff
@@ -0,0 +1,42 @@
+# CONTRACTS_V2_LOG
```

Comment:
does keeping this log make much sense? isn't a diff of master - v2.0.0 the same thing?

Perhaps we should start logging *integration changes* as a consequence of moving to v2, as per @boolafish ?

Reply:
Yess, sounds good! Lets remove the PR list part if we don't want it and increase upon the integrations section? This was just template-ish